AdyenCard/Components/GiftCardComponent/GiftCardComponent.swift

```diff
@@ -266,7 +265,7 @@ public final class GiftCardComponent: PartialPaymentComponent,
     }
 
     private func handle(orderResult: Result<PartialPaymentOrder, Swift.Error>) {
-        orderResult.handleError { delegate?.didFail(with: $0, from: self) }.handleSuccess(submit(order:))
+        orderResult.handle(submit(order:)) { delegate?.didFail(with: $0, from: self) }
```

Comment:
if there are 2 closures as a parameter, it might be better to not use trailing closure synthax, since we can't tel what the other closure is doing